src/components/NavBar.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const NavBar = ({ className }) => {
+  return (
+    <>
+      <div className={className}>
+        <div>
+          <div style={{marginRight:7, color:'white'}}>Home</div>
+          <input type="text" placeholder="Search" />
+        </div>
+
+        <div >
+          <div style={{marginRight:7, color:'white'}}>
+            <input id="modoVista" type="checkbox" name="modoVista" />
+            <label htmlFor="modoVista">DarkMode</label>
+          </div>
+          <div style={{color:'white'}}>Login</div>
+        </div>
+      </div>
+    </>
+  );
+};
+
+const LightNavBar = styled(NavBar)`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    background-color: #1c5476;
+    padding: 10px;
+
+    @media screen and (min-width: 430px) {
+        flex-direction: row;
+    }
+
+    @media screen and (max-width: 430px) {
+        flex-direction: column;
+    }
+
+    > div {
+        display: flex;
+        justify-content: space-around;
+    }
+`;
+
+const DarkNavBar = styled(NavBar)`
+    display: flex;
+    justify-content: space-between;
+    background-color: #556cd6;
+    padding: 10;
+
+    > div {
+        display: flex;
+        justify-content: space-around;
+    }
+`;
+
+export default {DarkNavBar, LightNavBar};
```

Comment:
Here you can do a styled component as well for `<div style={{marginRight:7, color:'white'}}`, it could be a single reusable Container component which you can customize the margin and color using props I think 🤔